### ISSUE TITLE
Make "alpha" status more prominent?

### ISSUE THREAD

**Issue by matklad** [intellij-rust/intellij-rust.github.io#2]
I think we should make it more clear that the plugin is currently in "alpha" and that it will be like this for a while. Perhaps just say "alpha" on the main page, and then add an explanation of what exactly does "alpha" means.

For me, alpha means:

* very far from the level of Scala/Java/Go support of IntelliJ.
* possible performance problems / crashes. 
* but already usable as the main Rust editor. I hack on Cargo and some pet projects using intelliJ Rust. Navigation is just awesome, editing is okish (cool formatter, poor completion, parser recovery, no niceties like postfix templates), working through compiler errors is meh (we need 100% accurate resolve to start adding red squiggles :( )

Basically, try it and use it, if you like it, or wait for the official release otherwise :)
